src/features/labels/links/PersonLink.tsx

```diff
@@ -56,47 +59,48 @@ export default function PersonLink({
     (admin) => admin.person.actor_id === person.actor_id,
   );
   const { hideUsernames } = useContext(ShareImageContext);
+  const currentUserHandle = useAppSelector(userHandleSelector);
 
   const onCommunityLinkLongPress = useCallback(() => {
     const state = store.getState();
-
     const blocks = state.site.response?.my_user?.person_blocks;
     const isBlocked = blocks?.some(
       (b) => getHandle(b.target) === getHandle(person),
     );
 
     stopIonicTapClick();
 
-    presentActionSheet({
-      cssClass: "left-align-buttons",
-      buttons: [
-        {
-          text: `${isBlocked ? "Unblock" : "Block"} User`,
-          icon: removeCircleOutline,
-          role: "destructive",
-          handler: () => {
-            (async () => {
-              try {
-                await dispatch(blockUser(!isBlocked, person.id));
-              } catch (error) {
-                presentToast({
-                  color: "danger",
-                  message: getBlockUserErrorMessage(error, person),
-                });
-                throw error;
-              }
-
-              presentToast(buildBlocked(!isBlocked, getHandle(person)));
-            })();
+    const isCurrentUser = currentUserHandle === getRemoteHandle(person);
+    const buttons: Button[] = [
+      isCurrentUser
+        ? "That's me!"
```

Comment:
Instead of this, can we hide the button? I realize it will just have the cancel button. However in the future if options are added this may be OK.

The alternative is to disable long press completely, or provide an alert saying you can't block yourself when you click block user.

Reply:
I'll go with removing the button for now